federation-2/router-bridge/src/planner.rs

```diff
@@ -651,6 +663,7 @@ impl Default for QueryPlannerConfig {
                 enable_defer: Some(false),
             }),
             graphql_validation: true,
+            reuse_query_fragments: None,
```

Comment:
if default is true -> shouldn't this be `Some(true)`?

Reply:
technically it shouldn't matter as when it is undefined we treat it as `true` (in JS code)